keps/sig-apps/1847-autoremove-statefulset-pvcs/kep.yaml

```diff
@@ -36,6 +36,5 @@ feature-gates:
       - kube-apiserver
 disable-supported: true
 
-# The following PRR answers are required at beta release
 # metrics:
-# Currently no metrics is planned for alpha release.
+# No new metrics are planned (the stateful set controller does not export any metrics).
```

Comment:
I'd encourage to add metrics, especially if none exist, yet

Reply:
Is this a hard requirement? Since nothing exists at this point, does this mean we're not promoting to beta in 1.25, or if I list metrics here that get added before the 1.25 freeze is that enough?

Reply:
Yes it is.